spiceypy/tests/test_wrapper.py

```diff
@@ -47,6 +47,22 @@ def setup_module(module):
     download_kernels()
 
 
+def test_threading_lock():
```

Comment:
It would be great to add additional tests for the enable/disable methods and no_threading_lock to improve the codcov report, preferably as additional `def test_...` functions, but they don't need to be to as complicated as this test (you could just use a simple spice function like b1900 to avoid loading kernels etc)